main.go

```diff
@@ -75,6 +75,11 @@ func setupCollectors(logger log.Logger, configFile string) ([]prometheus.Collect
 			vpcSessions = append(vpcSessions, sess)
 		}
 		vpcExporter := pkg.NewVPCExporter(vpcSessions, logger, config.VpcConfig, awsAccountId)
+
+		roleARN := os.Getenv("ROLE_ARN")
```

Comment:
I think this must be run before line 64, cause this is where the session is created